src/plugins_types.h

```diff
@@ -213,13 +213,14 @@ struct lysc_type_leafref;
  * @param[in] vecode Validity error code in case of LY_EVALID error code.
  * @param[in] path Path to the node causing the error.
  * @param[in] apptag Error-app-tag value.
- * @param[in] err_msg error formating message.
+ * @param[in] err_format Format string (same like at printf) or string literal.
```

Comment:
Mention the danger of using unknown variable (not in direct control of the caller) as format.